apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -281,6 +281,17 @@ defmodule OMG.ChildChain.BlockQueue.Core do
     :ok
   end
 
+  # https://github.com/ethereum/go-ethereum/blob/v1.9.15/core/tx_pool.go#L62-L64
+  def process_submit_result(
```

Comment:
I'm not entirely sure why do we even bother "handling" results. We don't do anything with them, we're just spending effort defining all function headers and then we ... log and return :ok?
I've never understood the purpose.

Reply:
I was wondering the same. It turns about the block queue uses these :ok, {:error, ...} returns to determine if the submission was successful or if it needs to retry the submission :(

I don't like it too but it's not a small refactor. I'm kind of unbundling this in https://github.com/omgnetwork/elixir-omg/pull/1575 though